src/server/scripts/Northrend/IcecrownCitadel/instance_icecrown_citadel.cpp

```diff
@@ -1319,28 +1321,31 @@ class instance_icecrown_citadel : public InstanceMapScript
             {
                 data << HeroicAttempts << ' '
                     << ColdflameJetsState << ' '
+                    << UpperSpireTeleporterActiveState << ' '
+                    << SisterSvalnaState << ' '
                     << BloodQuickeningState << ' '
-                    << BloodQuickeningMinutes << ' '
-                    << UpperSpireTeleporterActiveState;
+                    << BloodQuickeningMinutes;
             }
 
             void ReadSaveDataMore(std::istringstream& data) override
             {
+                uint32 temp = 0;
+
                 data >> HeroicAttempts;
 
-                uint32 temp = 0;
                 data >> temp;
-                if (temp == IN_PROGRESS)
-                    ColdflameJetsState = NOT_STARTED;
-                else
-                    ColdflameJetsState = temp ? DONE : NOT_STARTED;
+                (temp == DONE) ? ColdflameJetsState = DONE : ColdflameJetsState = NOT_STARTED;
```

Comment:
No.

Reply:
I get the general concept of what you are saying, better leave operations outside ternary operators, but, could you please elaborate on why, specifically?

like, is it cause of some undesired or missing or whatever compiler optimization? :)

Reply:
I honestly prefer if/else on multiple lines so I can put a breakpoint on the statement I want

Reply:
I agree on that too, but since the operations are rather simple, the break would only make sense to know the actual `temp` value...
and that's done a line before, right?